docs/output.md

```diff
@@ -1,57 +1,386 @@
 
-[FastQC](http://www.bioinformatics.babraham.ac.uk/projects/fastqc/) gives general quality metrics about your sequenced reads. It provides information about the quality score distribution across your reads, per base sequence content (%A/T/G/C), adapter contamination and overrepresented sequences.
+<details markdown="1">
+<summary>Output files</summary>
 
-For further reading and documentation see the [FastQC help pages](http://www.bioinformatics.babraham.ac.uk/projects/fastqc/Help/).
+- `smoothxg/`
+  - `<INPUT_FASTA>.smoothxg.gfa`: Smoothed pangenome graph in GFA format.
+  - `<INPUT_FASTA>.community.[0-9]{1,}.smoothxg.gfa`: Community GFA file containing the smoothed pangenome graph of the specific community. _Only appears when `--communities` is provided._
+  </details>
 
-**Output files:**
+## gfaffix
 
-* `fastqc/`
-  * `*_fastqc.html`: FastQC report containing quality metrics for your untrimmed raw fastq files.
-* `fastqc/zips/`
-  * `*_fastqc.zip`: Zip archive containing the FastQC report, tab-delimited data file and plot images.
+[gfaffix](https://github.com/marschall-lab/GFAffix) identifies walk-preserving shared affixes in variation graphs and collapses them into a non-redundant graph structure.
 
-> **NB:** The FastQC plots displayed in the MultiQC report shows _untrimmed_ reads. They may contain adapter sequence and potentially regions with low quality.
+<details markdown="1">
+<summary>Output files</summary>
+
+- `gfaffix/`
+  - `<INPUT_FASTA>.gfaffix.gfa`: Non-node-redundant pangenome graph in GFA format.
+  - `<INPUT_FASTA>.gfaffix.txt`: Graph shared affixes in TSV format.
+  - `<INPUT_FASTA>.community.[0-9]{1,}.gfaffix.gfa`: Community GFA file containing the non-node-redundant pangenome graph of the specific community. _Only appears when `--communities` is provided._
+  - `<INPUT_FASTA>.community.[0-9]{1,}.gfaffix.txt`: Community TSV file containing the graph shared affixes in TSV format of the specific community. _Only appears when `--communities` is provided._
+  </details>
+
+## odgi
+
+[odgi](https://github.com/pangenome/odgi) provides an efficient and succinct dynamic DNA sequence graph model, as well as a host of algorithms that allow the use of such graphs in bioinformatic analyses. In this pipeline, a huge variety of odgi's subcommands are used to process the built graphs. As a rule of thump, all files in ODGI format end with `.og`.
+
+### odgi build
+
+[odgi build](https://odgi.readthedocs.io/en/latest/rst/commands/odgi_build.html) constructs a dynamic succinct variation graph in ODGI format from a GFAv1.
+
+<details markdown="1">
+<summary>Output files</summary>
+
+- `odgi_build/`
+  - `<INPUT_FASTA>.*.og`: Graph in ODGI format.
+  - `<INPUT_FASTA>.community.[0-9]{1,}.*.og`: Community ODGI file of the specific community. _Only appears when `--communities` is provided._
+  </details>
+
+### odgi stats
```

Comment:
What are the two samples in the ODGI MultiQC reports, DRB1-3123 vs seqwish?

Reply:
The `seqwish` sample is the graph which was produced by `seqwish`.
The named sample, which just contains the sample name in the name.... is the final graph. 
`seqwish` -> `smoothxg` -> `gfaffix` -> `odgi build` -> `odgi unchop` -> `odgi sort`

Shall I add this to the Docs?

Reply:
I will push something tomorrow.

Reply:
Done in the `output.md`. If people still stumble about this later, I can still add more Docs.